### DISCUSSION TITLE
Can not upload a image after upgrade

### DISCUSSION THREAD

**Discussion by opravil-jan** [LycheeOrg/Lychee#2550] [Bugs]
### Lychee version

v5.5.1

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

When I try upload a image after upgrade to new version I'm getting error "stream_copy_to_stream(): Read of 8192 bytes failed with errno=21 Is a directory"

Setting up LIVEWIRE_ENABLED to false does not worked for me

LIVEWIRE_ENABLED=true
![image](https://github.com/user-attachments/assets/4666aadd-c246-4b77-8568-7689f75aaba0)
![image](https://github.com/user-attachments/assets/fc70ade2-76a4-4014-b9bc-8feebf687fb3)
![image](https://github.com/user-attachments/assets/f1b5a249-990d-4986-b15d-4a5eaf23a08b)

with 
LIVEWIRE_ENABLED=false

![image](https://github.com/user-attachments/assets/5ee96e33-0403-427f-b22b-fc83e2087371)

### Steps to reproduce the issue

Latest docker image of Lychee

docker-compose.yaml
version: '3.8'

services:
  lychee:
    image: lycheeorg/lychee
    container_name: lychee
    volumes:
      - ../../../../storage/conf:/conf
      - ../../../../storage/uploads:/uploads
      - ../../../../storage/sym:/sym
      - ../../../../storage/logs:/logs
#    command: "php artisan cache:table"
    environment:
      APP_NAME: Lychee
      APP_ENV: production
      APP_KEY: 
      APP_DEBUG: "true"
      # This MUST contain the host name up to the Top Level Domain (tld) e.g. .com, .org etc.
      APP_URL: "https://www.petr-bures.com"
      APP_FORCE_HTTPS: "yes"
      DEBUGBAR_ENABLED: "true"
      LIVEWIRE_ENABLED: "true"
      LOG_VIEWER_ENABLED: "true"
      LEGACY_V4_REDIRECT: "false"
      DB_OLD_LYCHEE_PREFIX: 
      DB_CONNECTION: pgsql
      DB_HOST: xxxx
      DB_PORT: 5432
      DB_DATABASE: xxxx
      DB_USERNAME: xxxx
      DB_PASSWORD: "xxxx"
      DB_LOG_SQL: "false"
      DB_LOG_SQL_EXPLAIN: "false" #only for MySQL
      DB_LIST_FOREIGN_KEYS: "false"
      TIMEZONE: Europe/Prague
      CACHE_DRIVER: redis
      SESSION_DRIVER: redis
      SESSION_LIFETIME: 120
      QUEUE_CONNECTION: sync
      SECURITY_HEADER_HSTS_ENABLE: "false"
      SECURITY_HEADER_CSP_CONNECT_SRC: 
      SECURITY_HEADER_SCRIPT_SRC_ALLOW: 
      REDIS_SCHEME: "tcp"
      REDIS_HOST: xxxx
      REDIS_PASSWORD: null
      REDIS_PORT: 6379
      TRUSTED_PROXIES: "*"
      SKIP_PERMISSIONS_CHECKS: "yes"
      STARTUP_DELAY: 10
      ADMIN_USER: bures.petr
      ADMIN_PASSWORD: xxxx
    restart: unless-stopped

networks: 
    default: 
      name: container-network
      external: true

### Diagnostics [REQUIRED]

Diagnostics
    -----------
    Warning: Dropbox import not working. dropbox_key is empty.
    Warning: zend.assertions is disabled although Lychee is in debug mode. For easier debugging code generation for assertions should be enabled.
    Info: Latest version of PHP is 8.3
    Warning: git (software) is not available.
    Info: Found 4 medium that could be generated.
         You can use `php artisan lychee:generate_thumbs medium 4` to generate them.

    System Information
    ------------------
    Lychee Version (tag):                    v5.5.1 (656ffae) -- 1 tags behind v6.0.0-alpha1 (-0.00012833911891571 years ago)
    DB Version:                              5.5.1
    
    composer install:                        --no-dev
    APP_ENV:                                 production
    APP_DEBUG:                               true
    APP_URL:                                 set
    APP_DIR:                                 default
    LOG_VIEWER_ENABLED:                      true
    LIVEWIRE_ENABLED:                        false
    
    System:                                  Linux
    PHP Version:                             8.2.20
    PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
    Timezone:                                Europe/Prague
    Max uploaded file size:                  100M
    Max post size:                           100M
    Livewire chunk size:                     12.00 MB
    Max execution time:                      3600
    PostgreSQL Version:                      PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
    
    exec() Available:                        yes
    Imagick Available:                       1
    Imagick Enabled:                         1
    Imagick Version:                         1691
    GD Version:                              2.3.3
    Number of foreign key:                   12 found.

    Config Information
    ------------------
    version:                                 050501
    check_for_updates:                       0
    sorting_photos_col:                      title
    sorting_photos_order:                    ASC
    sorting_albums_col:                      description
    sorting_albums_order:                    DESC
    imagick:                                 1
    skip_duplicates:                         0
    small_max_width:                         0
    small_max_height:                        360
    medium_max_width:                        1920
    medium_max_height:                       1080
    lang:                                    cz
    image_overlay_type:                      desc
    default_license:                         reserved
    compression_quality:                     90
    grants_full_photo_access:                1
    delete_imported:                         0
    mod_frame_enabled:                       1
    mod_frame_refresh:                       30
    thumb_2x:                                1
    small_2x:                                1
    medium_2x:                               1
    landing_page_enable:                     1
    force_migration_in_production:           0
    search_public:                           0
    hide_version_number:                     0
    SL_enable:                               1
    SL_for_admin:                            1
    recent_age:                              1
    grants_download:                         0
    photos_wraparound:                       1
    map_display:                             1
    zip64:                                   1
    map_display_public:                      1
    map_provider:                            Wikimedia
    force_32bit_ids:                         0
    map_include_subalbums:                   1
    update_check_every_days:                 3
    has_exiftool:                            1
    share_button_visible:                    1
    has_ffmpeg:                              1
    import_via_symlink:                      0
    location_decoding:                       1
    location_decoding_timeout:               30
    location_show:                           1
    location_show_public:                    1
    rss_enable:                              0
    rss_recent_days:                         7
    rss_max_items:                           100
    prefer_available_xmp_metadata:           0
    editor_enabled:                          1
    lossless_optimization:                   0
    swipe_tolerance_x:                       150
    swipe_tolerance_y:                       250
    local_takestamp_video_formats:           .avi|.mov
    log_max_num_line:                        1000
    unlock_password_photos_with_url_param:   0
    nsfw_visible:                            1
    nsfw_blur:                               0
    nsfw_warning:                            0
    nsfw_warning_admin:                      0
    nsfw_banner_override:                    
    map_display_direction:                   1
    album_subtitle_type:                     oldstyle
    upload_processing_limit:                 4
    new_photos_notification:                 0
    legacy_id_redirection:                   1
    zip_deflate_level:                       6
    default_album_protection:                1
    album_decoration:                        layers
    album_decoration_orientation:            row
    allow_username_change:                   1
    auto_fix_orientation:                    1
    use_job_queues:                          0
    random_album_id:                         starred
    use_last_modified_date_when_no_exif_date: 0
    layout:                                  square
    date_format_photo_thumb:                 M j, Y, g:i:s A e
    date_format_photo_overlay:               M j, Y, g:i:s A e
    date_format_sidebar_uploaded:            M j, Y, g:i:s A e
    date_format_sidebar_taken_at:            M j, Y, g:i:s A e
    date_format_hero_min_max:                F Y
    date_format_hero_created_at:             M j, Y, g:i:s A T
    date_format_album_thumb:                 M Y
    upload_chunk_size:                       0
    nsfw_banner_blur_backdrop:               0
    search_pagination_limit:                 1000
    search_minimum_length_required:          4
    photo_layout_justified_row_height:       320
    photo_layout_masonry_column_width:       300
    photo_layout_grid_column_width:          250
    photo_layout_square_column_width:        200
    photo_layout_gap:                        12
    display_thumb_album_overlay:             always
    display_thumb_photo_overlay:             hover
    default_album_thumb_aspect_ratio:        1/1
    use_album_compact_header:                0
    login_button_position:                   left
    back_button_enabled:                     0
    back_button_text:                        Return to Home
    current_job_processing_visible:          1
    SA_random_thumbs:                        0
    job_processing_queue_visible:            1
    login_required:                          0
    enable_unsorted:                         1
    enable_starred:                          1
    enable_recent:                           1
    enable_on_this_day:                      1
    login_required_root_only:                1

### Browser & System [REQUIRED]

Firefox 130, Fedora 40

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment]
@opravil-jan 

This seems to be a problem of space in your container, and not limited to version 5.
I suspect that the temporary folder of Lychee is being full.

I tried to provide a mitigation to that problem on version 6.
https://github.com/LycheeOrg/Lychee-Docker/blob/master/docker-compose.yml

Did you try restarting the container?
